pdns/dnsdistdist/dnsdist.service.in

```diff
@@ -22,7 +22,7 @@ StartLimitInterval=0
 TasksMax=8192
 LimitNOFILE=16384
 # Note: increasing the amount of lockable memory is required to use eBPF support
-# LimitMEMLOCK=infinity
+LimitMEMLOCK=infinity
```

Comment:
I have been torn about setting this by default for a long time, I wonder if there is any drawback. Can we somehow consume too many locked memory in a way that would impact the system's stability?
To be clear this is a general question that the author of this PR does not need to have an answer to :)

Reply:
I might be a bit conservative, but I do not like a process to be able to consume a resource without limit, especially if those resources are real (physical mem pages in this case) and not virtual. If we have some idea about the actual number needed, we could use that, say multiplied by a factor for future growth.

Reply:
That makes sense to me. So, Yogesh, if that's fine by you I suggest that you revert that change to keep the line commented for now and I'll open a new pull request with a more sensible value after doing some tests.

Reply:
I agree that not limiting resources to processes can have serious consequences. I'll keep this commented as for now.